### ISSUE TITLE
Build failed for mesos with version>=0.27.0

### ISSUE THREAD

**Issue by KinglyWayne** [mesos/hadoop#69]
with mesos 0.27.0 
Class org.apache.mesos.Protos.CommandInfo.ContainerInfo is removed (deprecated before 0.27.0)
as a result, mesos-hadoop build faild with mesos version >=0.27
mesos-hadoop code update is required to match new mesos version.
Thank you very much

**Comment by ZLhhs** [comment]
Yes, I try to build Hadoop 2.7.2 on Mesos 1.4.1 and when do "mvn package" commend , 
it returns : " /src/main/java/org/apache/mesos/hadoop/Utils.java:[46,28] cannot find symbol [ERROR] symbol:  class ContainerInfo".
